p2p/net/conn.go

```diff
@@ -119,9 +122,11 @@ func newConnection(conn readWriteCloseAddresser, netw networker,
 		deadliner:    conn,
 		networker:    netw,
 		session:      session,
+		messages:     make(chan []byte, MessageQueueSize),
+		stopSending:  make(chan bool),
 		msgSizeLimit: msgSizeLimit,
 	}
-
+	go connection.sendListener()
```

Comment:
maybe put this inside `beginEventProcessing` ? and before stating jut use `SendSock`